packages/ui/src/SelectNetwork/SelectNetwork.tsx

```diff
@@ -34,7 +34,7 @@ export const SelectNetwork: React.FC<SelectNetworkProps> = ({
 }) => (
   // TODO: add darkTheme icon or make sure icon work for both dark and light theme
   <Select {...props} className={className} items={items} aria-label="Select network" label="">
-    {({ chainId, src, label }) => (
+    {({ chainId, src, label }: SelectNetworkItem) => (
```

Comment:
Is this component actually still used now we have a new network selector in curve-ui-kit?